src/components/auth/RestrictedPage.tsx

```diff
@@ -0,0 +1,22 @@
+import { Square, Text, VStack } from '@chakra-ui/react'
+import { WarningIcon } from '@chakra-ui/icons'
+import Head from 'next/head'
+import React from 'react'
+
+const RestrictedPage = () => {
+  return (
+    <>
+      <Head>
```

Comment:
This component looks good. Here are some to-dos for this page

- [ ] A page should be found under the `/page` directory. If you want, you can treat it as a page and put it under the corresponding directory, add a button redirecting the user back to the home page should they not have the required access level.
- [ ] Alternatively, you may treat it as a component. If this is so, do remove the `<Head></Head>` and rename the function correspondingly.